api/include/opentelemetry/trace/trace_state.h

```diff
@@ -265,35 +181,37 @@ class TraceState
    */
   nostd::shared_ptr<TraceState> Delete(const nostd::string_view &key)
   {
-    nostd::shared_ptr<TraceState> ts{new TraceState()};
-
     if (!IsValidKey(key))
     {
-      return ts;
+      return TraceState::GetDefault();
```

Comment:
Earlier this was returning same trace_state instance. Why are we returning default now ?

Reply:
Earlier also it is returning empty trace state, right ? We are creating new instance and returning it. Am I missing anything here ?

Reply:
https://github.com/open-telemetry/opentelemetry-cpp/blob/main/api/include/opentelemetry/trace/trace_state.h#L270-L273
Maybe I would have done it wrong in my draft PR ( if taken from there), but existing code does return same trace_state instance. Which actually makes sense too.

Reply:
The trace_state returned is empty, right ? I am also taking reference from the same code. The ts created is empty.

Also,  expected behaviour is captured in the comments.
@returns empty TraceState object if key is invalid

Reply:
ok yeah, my bad :)